assets/javascripts/running.js

```diff
@@ -574,12 +575,18 @@ function updateDeveloperPanel() {
             statusAppendix = 'currently at: ' + developerMode.currentModule;
             if (developerMode.currentApiFunction) {
                 statusAppendix += ', ' + developerMode.currentApiFunction;
+                if (developerMode.currentApiFunctionArgs) {
+                    statusAppendix += ' ' + developerMode.currentApiFunctionArgs;
```

Comment:
I guess `developerMode.currentApiFunctionArgs` needs to be joined if it is an array, like in the other occurrence.

Reply:
JavaScript will happily do that by itself. Top example is from Chromium, bottom from Firefox.
![assert_screen](https://user-images.githubusercontent.com/53822705/63601517-e011c180-c5c5-11e9-9a1d-13de394624f6.png)